Makefile

```diff
@@ -228,8 +224,7 @@ govet:
 # Continuous integration targets
 
 ci: kops nodeup-gocode examples test govet
-	echo "Done"
-
+	sh -c hack/verify-boilerplate.sh
```

Comment:
I think I would prefer a verify-boilerplate target (and keeping the done message, but I'll just send a PR)